app/Http/Controllers/Api/V1/ProjectController.php

```diff
@@ -221,12 +234,14 @@ public function recent(Organization $suborganization)
      *
      * @urlParam required Organization $suborganization. Example: 1
```

Comment:
Remove "$" from "$videoId" and other params.

Reply:
Done!